src/index.html

```diff
@@ -24,8 +24,78 @@
         <link rel="preconnect" href="https://connect.facebook.net" crossorigin="anonymous">
         <link rel="prefetch" href="<%= process.env.API_ORIGIN %>/apps/cms/api/books/?format=json">
         <link rel="prefetch" href="<%= process.env.API_ORIGIN %>/apps/cms/api/footer/?format=json">
-        <link rel="stylesheet" href="https://ram.openstax.org/osano/osano.css">
-        <script src="https://cmp.osano.com/AzZqbXTbzhHsU3cv1/68d8e8ae-4024-4489-a000-72766ad284a6/osano.js"></script>
+        <script>
+          fetch('/accounts/api/user', { credentials: 'include' }).then((userResponse) => {
```

Comment:
osweb also fetches the user, is this gonna make it fetch the user twice on every page load?

Reply:
Yes... I can have both this and the app code write the user to the window (only if not already present), I suppose